lib/Translation/ExportVerilog/ExportVerilogFIRRTL.cpp

```diff
@@ -1340,6 +1341,43 @@ void ModuleEmitter::emitStatement(ConnectOp op) {
   emitLocationInfoAndNewLine(ops);
 }
 
+void ModuleEmitter::emitStatement(PartialConnectOp op) {
```

Comment:
Should this path also add bit select ops in the emitted Verilog or can this rely on Verilog truncation? 

I'm also noting that I hate the fact that PRs like this have to hit both paths and duplicate a bunch of logic that implements, effectively, the same behavior...

Reply:
I'm calling this path dead, unless someone speaks up in support of it.

Reply:
Please keep the ExportVerilogFIRRTL.cpp path around a little longer, but you don't have to give it a lot of love.  I would like to use it as a reference for whenever I get around to doing the implicit-width work and other general verilog cleanups.